addon/components/item-picker/item-row/multiple/template.hbs

```diff
@@ -5,7 +5,7 @@
     <span></span>
   </div>
   <div class="item-picker-item-results-item-inner">
-    <h2>
+    <h2 title="{{model.title}}">
```

Comment:
The PR title and description indicates that we are removing tooltips. But adding a title attribute to an element will cause most browsers to add their own tooltip (it won't look like our app's bootstrap tooltips).

Reply:
I'm not 100% sure. I pinged @samanthahunter on Slack to confirm.

Reply:
Confirmed with @samanthahunter that the title should be removed as well